images/outputs.tf

```diff
@@ -1,3 +1,13 @@
+output "administerkmskeys_role_arn" {
```

Comment:
Again, I'm wondering if we want a role that can administer _all_ KMS keys, or just the one used to encrypt AMIs.

If the former, then the description should state "...to administer all KMS keys in the ..." for clarity.

If the latter then the description should state "...to administer the KMS key used to encrypt AMIs in the..." for clarity.

These changes should roll up into the README as well.

Reply:
As discussed, we are going with administration of all KMS keys in the account.  Clarified in https://github.com/cisagov/cool-accounts/pull/17/commits/97ad10252e39d2b4ea8398a827e5da0aa5453e3e.